app/src/main/res/layout/quicksettings_permissions.xml

```diff
@@ -137,4 +137,29 @@
         tools:text="@string/preference_option_phone_feature_blocked"
         tools:visibility="visible" />
 
+    <TextView
+        android:id="@+id/mediaKeySystemAccessLabel"
+        style="@style/QuickSettingsText.Icon"
+        android:layout_width="0dp"
+        android:layout_height="@dimen/quicksettings_item_height"
+        app:drawableStartCompat="@drawable/ic_link"
+        android:text="@string/preference_phone_feature_media_key_system_access"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/mediaKeySystemAccessStatus"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:visibility="visible" />
+
+    <TextView
+        android:id="@+id/mediaKeySystemAccessStatus"
+        style="@style/QuickSettingsText.PermissionItemEnd"
+        android:layout_width="wrap_content"
+        android:layout_height="@dimen/quicksettings_item_height"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/mediaKeySystemAccessLabel"
+        tools:text="@string/preference_option_phone_feature_blocked"
+        tools:visibility="visible" />
+
```

Comment:
There is an issue here we are overlapping the content with the location permission  

![image](https://user-images.githubusercontent.com/773158/101199290-aa65e000-3632-11eb-97f6-f81165eec823.png)

Reply:
I think we need to update the values of the previous last item `persistentStorage`

![image](https://user-images.githubusercontent.com/773158/101203078-27e01f00-3638-11eb-897f-b1851bf98d57.png)

Reply:
Nice catch! I found all the pieces of UI but this one, and of course I goofed it ;)

Fixed, thank you!